strictEqual.js

```diff
@@ -0,0 +1,10 @@
+const strictEqual = (valueOne, valueTwo) => {
+     // compare the value and type of value one to that of value two, returning true if both are equal
+
+     if (valueOne == valueTwo && typeof valueOne == typeof valueTwo) {
+        return true 
+     }           
+     return false
```

Comment:
Here we are saying if the condition is true return true, if the condition is false return false. That means we can just return the condition itself and eliminate the if.

```JavaScript
const strictEqual = (valueOne, valueTwo) => {
    return valueOne == valueTwo && typeof valueOne == typeof valueTwo
}
```

Since this would make it a one line return function we could use fat arrow's implicit return

```JavaScript
const strictEqual = (valueOne, valueTwo) => valueOne == valueTwo && typeof valueOne == typeof valueTwo
```